pkg/cmd/pr/merge/merge.go

```diff
@@ -444,18 +467,25 @@ func NewMergeContext(opts *MergeOptions) (*mergeContext, error) {
 		return nil, err
 	}
 
+	apiClient := api.NewClientFromHTTP(httpClient)
+	repo, err := api.GitHubRepoWithMergeQueue(apiClient, baseRepo, pr.BaseRefName)
```

Comment:
`baseRepo` has some but not all of the fields that `repo` provides, which leads to this mostly duplicated api call as we already get the repo with `opts.Finder.Find`. This would be a great place to do a refactor and have all the needed fields in one place, but I wasn't sure how best to go about that here and would greatly appreciate feedback!